e2e/tests/ui-driven/src/helpers/geoSpatialUserActions.ts

```diff
@@ -0,0 +1,33 @@
+import { Page } from "@playwright/test";
+import { setupOSMockResponse } from "../mocks/osPlacesResponse";
+
+export async function answerFindProperty(page: Page) {
+  await setupOSMockResponse(page);
+  await page.getByLabel("Postcode").fill("SW1 1AA");
+  await page.getByLabel("Select an address").click();
+  await page.getByRole("option").first().click();
+}
+
+export const userChallengesPlanningConstraint = async (page: Page) => {
+  const thisDoesNotApplyConstraintButton = page.getByRole("button", {
+    name: "I don't think this constraint",
```

Comment:
nit: can we check the full button text here / this substring is a bit confusing? Eg `"I don't think this constraint applies to this property`"